File: engine/src/main/java/client/database/data/QuestData.java
package client.database.data;

public class QuestData {
   private short questId;

   private int status;

   private long time;

   private long expires;

   private int forfeited;

   private int completed;

   private int questStatusId;

   public QuestData(short questId, int status, long time, long expires, int forfeited, int completed, int questStatusId) {
      this.questId = questId;
      this.status = status;
      this.time = time;
      this.expires = expires;
      this.forfeited = forfeited;
      this.completed = completed;
      this.questStatusId = questStatusId;
   }

   public short getQuestId() {
      return questId;
   }

   public int getStatus() {
      return status;
   }

   public long getTime() {
      return time;
   }

   public long getExpires() {
      return expires;
   }

   public int getForfeited() {
      return forfeited;
   }

   public int getCompleted() {
      return completed;
   }

   public int getQuestStatusId() {
      return questStatusId;
   }
}

File: engine/model/src/main/scala/client/database/data/QuestData.scala
package client.database.data

class QuestData(private var _questId: Short, private var _status: Int, private var _time: Long,
                private var _expires: Long, private var _forfeited: Int, private var _completed: Int,
                private var _questStatusId: Int) {
  def questId: Short = _questId

  def status: Int = _status

  def time: Long = _time

  def expires: Long = _expires

  def forfeited: Int = _forfeited

  def completed: Int = _completed

  def questStatusId: Int = _questStatusId
}
